### PR TITLE
"Incognito mode" notes

### PR DESCRIPTION
**Pull request by ianbjacobs**
Responsive to:
https://github.com/w3c/browser-payment-api/issues/230

### CODE REVIEW

**Comment by rvm4** [comment]
This seems pretty reasonable and straight-forward. I cannot see anyone objecting to this language. I'm happy to merge this if no one speaks up to the contrary soon.

File: index.html [hunk 1]
```diff
@@ -1355,12 +1355,18 @@ <h2 id="state-transitions" class="informative">State transitions</h2>
...
     <p>
       The <a>user agent</a> should never share information about the user to the web page
       (such as the shipping address) without user consent.
     </p>
   </section>
+  <section>
+    <h3>Incognito Mode</h3>
+    <ul>
+      <li>This API should remain available in a browser's "incognito mode".</li>
+      <li>Browsers that offer "persistent permission" features for sharing user payment information automatically with merchants should ignore those settings in icognito mode and require explicit user consent.</li>
```
Comment (adrianhopebailie) [inline index.html:1368]: icognito -> incognito

**Comment by adrianba** [comment]
I think it's up to implementations to decide how to integrate features into their private browsing modes given the goals they have for those modes. The spec doesn't need to express a preference here. It probably also should not use feature names from particular browsers.

**Comment by adrianhopebailie** [comment]
Closed but reopening #230 to track this

**Comment by burdges** [comment]
We should however take seriously any observations that aspects of the payment spec seem problematic for private browsing mode.

**Pull request closed without merging**